src/main/java/com/sobytylnik/ProfileController.java

```diff
@@ -15,29 +14,37 @@
 public class ProfileController {
 
     @Autowired
-    @Qualifier("SQLProfileRepository")
-    private ProfileRepository repository;
+    private SQLSpringProfileRepository repository;
 
     @RequestMapping(method = RequestMethod.GET)
     public List<Profile> getAllProfiles(){
-        return repository.findAllProfiles();
+        List<Profile> profiles = new ArrayList<>();
+        repository.findAll().forEach(profiles::add);
```

Comment:
It's a bad practice to mutate some external state from Stream API.
It's better to collect into some object:
`List<Profile> profiles = repository.findAll().stream().collect(toList());`
But this is equal to a simple `List<Profile> profiles = repository.findAll()`.